classes/module/Module.php

```diff
@@ -340,6 +348,58 @@ public function __construct($name = null, Context $context = null)
         }
     }
 
+    private function registerBundles(ModuleManifestLoader $moduleManifestLoader)
+    {
+        $manifestValues = $moduleManifestLoader->getValues();
+
+        $configFile = _PS_CONFIG_DIR_ . DIRECTORY_SEPARATOR . 'module_bundles.php';
+        if (!file_exists($configFile)) {
+            $this->writeBundlesFile([], $configFile);
+        }
+
+        $registeredBundles = include $configFile;
+        $moduleBundles = [];
+        foreach ($manifestValues['bundles'] as $bundle) {
+            $moduleBundles[] = $bundle;
+        }
+        $registeredBundles[$this->name] = $moduleBundles;
+
+        $this->writeBundlesFile($registeredBundles, $configFile);
+    }
+
+    private function writeBundlesFile(array $moduleBundles, string $configFile)
+    {
+        $content = "<?php\nreturn [\n";
```

Comment:
check var_export

Reply:
I tried with var_export. Too complicated. 

The output I try to have is 
```
<?php
return [
	'ps_healthcheck' => [
		new PrestaShop\Module\HealthCheck\PsHealthcheckBundle(),
	],
];
```

If I take the instantiated bundle, I have an output like
```
<?php
return array (
  'ps_healthcheck' => 
  array (
    0 => 
    PrestaShopModuleHealthCheckPsHealthcheckBundle::__set_state(array(
       'name' => NULL,
       'extension' => NULL,
       'path' => NULL,
       'namespace' => NULL,
       'container' => NULL,
    )),
  ),
);
```

So a representation of the bundle object itself

If I set to the array the instantiation, 
`$modules[$moduleName][] = sprintf("new %s()", get_class($bundle));`

The output is 
```
<?php
return array (
  'ps_healthcheck' => 
  array (
    0 => 'new PrestaShop\Module\HealthCheck\PsHealthcheckBundle()',
  ),
);
```
It's a string not an instruction to create the bundle object

Reply:
the `set_state` is a magic method for PHP to restore the statement of a class, it allows you to export your class easily at a moment. (https://www.php.net/manual/en/language.oop5.magic.php#object.set-state) 
It should be enough :thinking: 

Otherwise you can store something like this:
```php
return [
  'ps_healtcheck' => [
    PsHealthcheckBundle::class,
  ],
];
```
But need an adaptation in the bundle runner et not sure it's the best idea :sweat_smile: